configureWorkspace/configure.ts

```diff
@@ -637,28 +675,35 @@ export async function configure(folderPath?: string): Promise<void> {
         return createWorkspaceFileIfNotExists(fileName, DOCKER_FILE_TYPES[fileName]);
     }));
 
-    if (reporter) {
-        /* __GDPR__
-        "command" : {
-            "command" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" },
-            "platformType": { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
-        }
-        */
-        reporter.sendTelemetryEvent('command', {
-            command: 'vscode-docker.configure',
-            platformType
-        });
-    }
+    // Don't wait
+    vscode.window.showInformationMessage(
+        filesWritten.length ?
+            `The following files were written into the workspace:${EOL}${EOL}${filesWritten.join(', ')}` :
+            "No files were written"
+    );
 
     async function createWorkspaceFileIfNotExists(fileName: string, generatorFunction: GeneratorFunction): Promise<void> {
-        const workspacePath = path.join(folderPath, fileName);
+        const workspacePath = path.join(rootFolderPath, fileName);
+        let writeFile = false;
         if (fs.existsSync(workspacePath)) {
-            const item: vscode.MessageItem = await vscode.window.showErrorMessage(`A ${fileName} already exists. Would you like to override it?`, ...YES_OR_NO_PROMPT);
-            if (item.title.toLowerCase() === 'yes') {
-                fs.writeFileSync(workspacePath, generatorFunction(serviceName, platformType, os, port, pkg), { encoding: 'utf8' });
+            const response: vscode.MessageItem = await vscode.window.showErrorMessage(`A ${fileName} already exists.Would you like to overwrite it ? `, ...YES_OR_NO_PROMPTS);
```

Comment:
nit: Seems like the whitespace changes to this message might've been an accident? Specifically the spaces around the punctuation

Also rather than `A ${filename} already exists.` I would prefer `"${filename}" already exists.`